service/client.go

```diff
@@ -0,0 +1,57 @@
+package service
+
+import (
+	"github.com/derekparker/delve/service/api"
+)
+
+// Client represents a debugger service client. All client methods are
+// synchronous.
+type Client interface {
+	// Detach detaches the debugger, optionally killing the process.
+	Detach(killProcess bool) error
+
+	// GetState returns the current debugger state.
+	GetState() (*api.DebuggerState, error)
+
+	// Continue resumes process execution.
+	Continue() (*api.DebuggerState, error)
+	// Next continues to the next source line, not entering function calls.
+	Next() (*api.DebuggerState, error)
+	// Step continues to the next source line, entering function calls.
```

Comment:
Change to:

`// Step continues for a single instruction, entering function calls.`